### DISCUSSION TITLE
Failed to install on macOS Big Sur 11.4

### DISCUSSION THREAD

**Discussion by babycaseny** [DraqueT/PolyGlot#986] [General]
I am using a MacBook Pro running macOS Big Sur 11.4 (just updated several days ago). I failed to run the installation program.

**Comment by DraqueT** [comment]
Thanks for reporting this! I'm curious, what exactly is happening? Does the installation program itself fail, then give an error message? Does PolyGlot itself fail to open once it's installed? If it's the second, it's likely OSX blocking the program's first open, as it's not licensed with Apple (this costs money). If that's the case, you can make it open by navigating to the app in Finder, right clicking, then clicking Open. It'll give you an "Are you sure?" dialog. Select yes and you should be good to go.

And if it's something else, I'll help you get past it as well. : )
